signer/local/testdata/inter_pathlen_0-key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
I realize you would never use the -key and cert.pem in those local signer tests, better not check them in. Just keep the generated around.